src/Operation/FindAndModify.php

```diff
@@ -153,6 +165,10 @@ public function __construct($databaseName, $collectionName, array $options)
             throw InvalidArgumentException::invalidType('"fields" option', $options['fields'], 'array or object');
         }
 
+        if (isset($options['hint']) && ! is_string($options['hint']) && ! is_array($options['hint']) && ! is_object($options['hint'])) {
+            throw InvalidArgumentException::invalidType('"hint" option', $options['hint'], 'string or array or object');
```

Comment:
With recent changes to `InvalidArgumentException::invalidType`, you can use the following:

```suggestion
            throw InvalidArgumentException::invalidType('"hint" option', $options['hint'], ['string', 'array', 'object']);
```

As an added bonus, this will also make use of proper Oxford comma grammar 😛